pkg/virt-operator/resource/apply/rbac_test.go

```diff
@@ -0,0 +1,338 @@
+package apply
```

Comment:
@iholder-redhat Can you please add a GPL header?

Reply:
Done. Thanks for noticing!